components/fxa-client/src/state_persistence.rs

```diff
@@ -280,7 +281,7 @@ mod tests {
     fn test_v2_ignores_unknown_fields_introduced_by_future_changes_to_the_schema() {
         // This is a snapshot of what some persisted StateV2 data would look before any backwards-compatible changes
         // were made. It's very important that you don't modify this string, which would defeat the point of the test!
```

Comment:
Per the comment here, it's very important that you don't modify this string. There will be clients in the wild with a string that looks like this and we don't want to accidentally break support for them.

(I guess technically *removing* fields from the string won't affect the stated purpose of the test, which is to check whether new fields get correctly default to none. But I think it violates the spirit of the test. You could consider copying it into another test named something like `test_v2_ignores_unknown_fields_removed_in_future_changes_to_the_schema` but I don't think we should just remove it entirely)